package.json

```diff
@@ -39,7 +38,12 @@
     "pagination"
   ],
   "homepage": "https://github.com/juanpablo-rojas/node-pagination#readme",
-  "dependencies": {},
+  "dependencies": {
+    "express-wolox-logger": "1.1.0",
+    "http": "0.0.0",
+    "umzug": "^2.1.0",
+    "faker": "^4.1.0"
```

Comment:
- `http` is a native module of node, not necessary to install as dependency
- `faker` and `umzug` should be devDependencies
-`express-wolox-logger` should be not necessary